smoltcp-device/src/lib.rs

```diff
@@ -0,0 +1,391 @@
+#![cfg_attr(not(any(test, feature = "std")), no_std)]
+#![deny(unsafe_code)]
+
+/*! Networking hardware abstraction.
+
+This crate deals with the *network devices*. It provides a trait
+for transmitting and receiving frames, [Device](trait.Device.html).
+*/
+#![cfg_attr(
+    feature = "provides-medium-ethernet",
+    doc = r##"
+# Examples
+
+An implementation of the [Device](trait.Device.html) trait for a simple hardware
+Ethernet controller could look as follows:
+
+```rust
+use smoltcp_device::{DeviceCapabilities, Device, Medium};
+use smoltcp::time::Instant;
```

Comment:
I think this should be `smoltcp_device`.

Reply:
Indeed, thank you